webapp/app/forms/steps/lotse/personal_data_steps.py

```diff
@@ -299,6 +300,31 @@ def __init__(self, **kwargs):
             template='lotse/form_person_a.html'
         )
 
+    @classmethod
+    def get_label(cls, data=None):
+        return ngettext('form.lotse.step_person_a.label', 'form.lotse.step_person_a.label',
+                        num=get_number_of_users(data))
+
+    def render(self, data, render_info):
+        number_of_users = get_number_of_users(data)
+        render_info.step_title = ngettext('form.lotse.person-a-title', 'form.lotse.person-a-title',
+                                          num=number_of_users)
+        render_info.step_intro = ngettext('form.lotse.person-a-intro', 'form.lotse.person-a-intro',
+                                          num=number_of_users)
```

Comment:
I think if you don't want a text here for the single-case, you could just leave the string empty:
```suggestion
        render_info.step_intro = ngettext('', 'form.lotse.person-a-intro',
                                          num=number_of_users)
```

Reply:
If we do this in combination with setting `intro=None` in the init, babel will no longer generate the corresponding message. Therefore, I would keep it that way?

Reply:
Ah yes, it makes sense that it worked for me because of the default! 👍